index.html

```diff
@@ -97,6 +97,35 @@ <h2>Connect</h2>
           <a href="www.linkedin.com/in/marilynmalpu">LinkedIn</a>
         </li>
       </ul>
+    </section id="message_from_section">
+    <h2>Leave a message</h2>
+    <form name="leave_message" class="message-form">
+      <label for="usersName">Name:</label>
+      <input
+      type="text"
+      id="usersName"
+      name="usersName"
+      required="true"
+      /><br />
+      <label for="usersEmail">Email:</label>
+      <input
+      type="email"
+      id="usersEmail"
+      name="usersEmail"
+      required="true"
+      /><br />
+      <label for="usersMessage">Message:</label>
+      <textarea
+      name="usersMessage"
+      id="usersMessage"
+      required="true"
+      ></textarea
+      ><br />
+      <button type="submit" value="submit" name="submit" class="submit-buttom">Submit</button>
```

Comment:
You don't need this attribute.

Reply:
when you say "you don't need.." means that I can delete and nothing is gonna happen?

Reply:
Yes, nothing should happen if you remove the `value` attribute.